XIVSlothCombo/Combos/PvE/MCH.cs

```diff
@@ -697,7 +733,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     if (IsEnabled(CustomComboPreset.MCH_Heatblast_Wildfire) &&
                         ActionReady(Hypercharge) &&
```

Comment:
Is this check necessary? As far as I can tell checking for both `ActionReady(Hypercharge) && WasLastAbility(Hypercharge)` will always return false. I had to remove `ActionReady(Hypercharge)` in my build to make this part work,

Reply:
should be levelcheck instead of actionready.